controllers/gslb_controller.go

```diff
@@ -195,13 +198,82 @@ func (r *GslbReconciler) SetupWithManager(mgr ctrl.Manager) error {
 			return nil
 		})
 
+	createGslbFromIngress := func(annotationKey string, annotationValue string, a handler.MapObject, strategy string) {
+		log.Info(fmt.Sprintf("Detected strategy annotation(%s:%s) on Ingress(%s)",
+			annotationKey, annotationValue, a.Meta.GetName()))
+		c := mgr.GetClient()
+		ingressToReuse := &v1beta1.Ingress{}
+		err := c.Get(context.Background(), client.ObjectKey{
+			Namespace: a.Meta.GetNamespace(),
+			Name:      a.Meta.GetName(),
+		}, ingressToReuse)
+		if err != nil {
+			log.Info(fmt.Sprintf("Ingress(%s) does not exist anymore. Skipping...", a.Meta.GetName()))
+			return
+		}
+		gslbExist := &k8gbv1beta1.Gslb{}
+		err = c.Get(context.Background(), client.ObjectKey{
+			Namespace: a.Meta.GetNamespace(),
+			Name:      a.Meta.GetName(),
+		}, gslbExist)
+		if err == nil {
+			log.Info(fmt.Sprintf("Gslb(%s) already exists. Skipping...", gslbExist.Name))
```

Comment:
Consider update for better context:
```suggestion
			log.Info(fmt.Sprintf("Gslb(%s) already exists. Skipping Gslb creation...", gslbExist.Name))
```